src/main/java/org/folio/rest/impl/RequestExpiryImpl.java

```diff
@@ -30,19 +30,19 @@ public void expireRequests(Map<String, String> okapiHeaders,
 
     Vertx vertx = context.owner();
     new ConfigurationClient(vertx, okapiHeaders).getTlrSettingsOrDefault()
-    .compose(tlrSettings -> createRequestExpirationService(okapiHeaders, vertx, tlrSettings)
+    .compose(tlrSettings -> createRequestExpirationService(okapiHeaders, vertx, tlrSettings, context)
```

Comment:
Object `vertx` comes from `context`. Should we remove it from method parameters and extract it from `context` within `createRequestExpirationService()`?